content/blog/understanding-hpe-greenlake-identity-access-management.md

```diff
@@ -0,0 +1,291 @@
+| **HPE GreenLake Billing**                       |                                                                                                                                                                      |     |
+| Billing Contributor                             | View billing information and edit downstream rates                                                                                                                   |     |
+| Billing Viewer                                  | View billing information                                                                                                                                             |     |
+| Billing Usage Viewer                            | View Monthly Charges card and report (usage information only - no cost information)                                                                                  |     |
+| **HPE GreenLake for Private Cloud**             |                                                                                                                                                                      |     |
+| Private Cloud Tenant Owner                      | Administer HPE GreenLake for private cloud dashboard<br>Manage scheduling and activity<br>Manage infrastructure<br>Manage provisioning                               |     |
+| Private Cloud Tenant Contributor                | Manage self-service VMs and app provisions                                                                                                                           |     |
+| **HPE GreenLake for Containers**                |                                                                                                                                                                      |     |
+| Container Platform Cluster Owner                | Manage predefined and custom cluster blueprints, and machine blueprints                                                                                              |     |
+| Container Platform Cluster Resource Contributor | Manage resources within a cluster namespace                                                                                                                          |     |
+| **HPE GreenLake for ML Ops**                    |                                                                                                                                                                      |     |
+| MLOps Admin and IAM Owner                       | Install services and manage projects, resources, and users, and view sites information                                                                               |     |
+| MLOps Project Admin                             | To build, train, and deploy models, launch the AI/ML services<br>Corresponds to Project Administrator role in HPE Ezmeral Container Platform                         |     |
```

Comment:
Please note that HPE Ezmeral Container Platform has recently been renamed to "HPE Ezmeral Runtime Enterprise".

Reply:
I found the text "HPE Ezmeral Container Platform" in the HPE GreenLake Central User Guide (https://support.hpe.com/hpesc/public/docDisplay?docId=a00092451en_us&docLocale=en_US&page=GUID-258784AD-AB97-4CAE-A893-5284329AD8E4.html).
If the new title is "HPE Ezmeral Runtime Enterprise" I can change it here but it probably means that it is incorrect elsewhere.